lhotse/recipes/slu.py

```diff
@@ -0,0 +1,124 @@
+import glob
+import json
+import logging
+from collections import defaultdict
+from itertools import groupby
+from pathlib import Path
+from typing import Dict, List, NamedTuple, Optional, Tuple, Union
+
+import pandas
```

Comment:
Please move pandas (and ideally also soundfile) into the local import scope so that importing this file does not trigger a hard requirement.

Reply:
Should be fixed in the latest commit! (Turns out soundfile was not used at all)